src/components/addNewBook.js

```diff
@@ -1,58 +1,93 @@
 import React, { useState } from 'react';
 import { useDispatch } from 'react-redux';
 import { v4 as uuidv } from 'uuid';
-import { addBook, removeBook } from '../redux/books/books';
-import store from '../redux/configureStore';
-import BookForm from './inputBook';
-import Books from './Books';
+import { addBook } from '../redux/books/books';
+// import store from '../redux/configureStore';
+// import BookForm from './inputBook';
+// import Books from './Books';
```

Comment:
[ OPTIONAL ] Please remove commented code from here as it can ultimately make the whole project unclean.